### ISSUE TITLE
Chart Animation Rendering stucks

### ISSUE THREAD

**Issue by Andries-Smit** [mendix/ChartJS#73]
Having Multiple Graphs on one page. The rendering works most of the time fine... however sometimes it stops the animation before it is finished.

![image](https://cloud.githubusercontent.com/assets/6724749/14183027/5aecc0aa-f76e-11e5-9e22-08e53d542c4b.png)

The option "Responsive animation rendering" is set to 0

**Comment by JelteMX** [comment]
That's weird, because it shouldnt animate when this is set to 0. In which Modeler version does this occur?

**Comment by Andries-Smit** [comment]
Mendix 6.3.1

Note that. It is a Data View containing 6 graphs, that listens to a List
View.

2016-04-01 10:26 GMT+02:00 Jelte Lagendijk notifications@github.com:

**Comment by Andries-Smit** [comment]
It could well be that there is an refresh/interrupt during the animation. 

![renderstackbargraph](https://cloud.githubusercontent.com/assets/6724749/14213074/51c11724-f836-11e5-8ab2-6fdeab1009d2.gif)

https://github.com/mendix/ChartJS/blob/master/src/ChartJS/widgets/StackedBarChart/widget/StackedBarChart.js#L146

**responsiveAnimationDuration**, is is not used for global animations

However the option **animation** does, but is not configurable.

Would be nice feature to add.

**Comment by JelteMX** [comment]
the option animation cannot be set to false (as in Chart.JS version 1), but I managed to set the animation duration to 0. I tested today with 10 charts on one page, responsiveAnimationDuration to 0 and animation  duration to 0, and that seems to do the trick

**Comment by Andries-Smit** [comment]
Animations are still moving... 

The sugested fix, is global and not function in my case, not sure why

Core.js:

```
if (!this.chartAnimation) {
    this._chartJS.defaults.global.animation.duration = 0;
}
```

A better solution will probably by setting it as an option in the individual charts

BarChart.js

```
options: this._chartOptions({
     animation:  this.chartAnimation,
```

This should be applied to all type of charts....

**Comment by JelteMX** [comment]
@Andries-Smit I was already one step ahead of you in the upcoming v3.5.0 release ;-)

https://github.com/mendix/ChartJS/blob/master/src/ChartJS/widgets/Core.js#L575

``` js
                animation: {
                    duration: this.chartAnimation ? this.animationDuration : 0
                }
```

As fixed in these commits: 4963660 & e6d8c2b
